components/common.yaml

```diff
@@ -17,8 +17,10 @@ schemas:
     type: string
     enum:
       - autoDetectedUserInstructions
+      - binaries
```

Comment:
Should these missing types also exit in `lib/types.ts` as well?

Reply:
Adding them to typescript would be misleading because snyk-docker-plugin doesn't actively use them.